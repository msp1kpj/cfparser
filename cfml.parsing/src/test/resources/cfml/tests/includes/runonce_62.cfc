include "mypage.cfm" runonce=true;