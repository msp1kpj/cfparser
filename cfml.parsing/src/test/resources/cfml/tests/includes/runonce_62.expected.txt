/*===TOKENS===*/
INCLUDE                  <include>
OPEN_STRING              <">
STRING_LITERAL           <mypage.cfm>
CLOSE_STRING             <">
IDENTIFIER               <runonce>
'='                      <=>
BOOLEAN_LITERAL          <true>
';'                      <;>
/*===TREE===*/
(scriptBlock
  (element
    (statement
      (tagOperatorStatement
        (includeStatement
          include
          (baseExpression
            (unaryExpression
              (primaryExpression
                (literalExpression (stringLiteral " (stringLiteralPart mypage.cfm) "))
              )
            )
          )
          (paramStatementAttributes
            (param
              (multipartIdentifier (identifier runonce))
              =
              (startExpression
                (compareExpression
                  (baseExpression (unaryExpression (primaryExpression (literalExpression true))))
                )
              )
            )
          )
          ;
        )
      )
    )
  )
)
/*======*/
/*===DECOMPILE===*/
include 'mypage.cfm'
/*======*/